sdk/eventhub/Azure.Messaging.EventHubs/src/EventData.cs

```diff
@@ -59,8 +58,11 @@ public class EventData
         /// </value>
         ///
         /// <remarks>
-        ///   The <see cref="ContentType" /> is intended to allow coordination between event
-        ///   producers and consumers.  It has no meaning to the Event Hubs service.
+        ///   The <see cref="ContentType" /> is managed by the application and is intended to
+        ///   allow coordination between event producers and consumers.
+        ///
```

Comment:
Does this spacing get rendered without the `<para>` element?

Reply:
Yup;   I fixed the doc tools for that one, personally.  😄

Reply:
https://github.com/docascode/ECMA2Yaml/pull/190